import DomainException from "./DomainExceptions.js";

class UserAlreadyExistsException extends DomainException {
  constructor(message = "L'utente esiste gi√†") {
    super(message, 409);
  }
}

export default UserAlreadyExistsException;
